torchgeo/datasets/landcoverai.py

```diff
@@ -2,31 +2,33 @@
 # Licensed under the MIT License.
 
 """LandCover.ai dataset."""
-
+import abc
 import glob
 import hashlib
 import os
 from functools import lru_cache
-from typing import Callable, Dict, Optional
+from typing import Any, Callable, Dict, List, Optional, cast
 
 import matplotlib.pyplot as plt
 import numpy as np
 import torch
 from matplotlib.colors import ListedColormap
 from PIL import Image
+from rasterio.crs import CRS
 from torch import Tensor
+from torch.utils.data import Dataset
 
-from .geo import NonGeoDataset
-from .utils import download_url, extract_archive, working_dir
+from .geo import NonGeoDataset, RasterDataset
+from .utils import BoundingBox, download_url, extract_archive, working_dir
 
 
-class LandCoverAI(NonGeoDataset):
-    r"""LandCover.ai dataset.
+class LandCoverAIBase(Dataset[Dict[str, Any]], abc.ABC):
+    r"""Abstract base class for LandCover.ai Geo and NonGeo datasets.
 
-    The `LandCover.ai <https://landcover.ai/>`__ (Land Cover from Aerial Imagery)
+    The `LandCover.ai <https://landcover.ai.linuxpolska.com/>`__ (Land Cover from Aerial Imagery)
```

Comment:
flake8 is complaining about this line being too long, can you wrap it onto the next line?

Reply:
Yeah, I missed that